src/main/java/org/broadinstitute/hellbender/tools/spark/sv/GATKSVVCFHeaderLines.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GATKSVVCFHeaderLines {
+
+    public static Map<String, VCFHeaderLine> vcfHeaderLines = new HashMap<>();
+
+    // VCF standard SV header lines
+    // todo: add these and the other standard SV info fields from the VCF spec to htsjdk VCFStandardHeaderLines
+    public static final String SVTYPE = "SVTYPE";
+    public static final String SVLEN = "SVLEN";
+
+    // GATK-SV specific header lines
+    public static final String TOTAL_MAPPINGS = "TOTAL_MAPPINGS";
```

Comment:
Yeah, I was just thinking about convenience (and the ability to apply filters to the VCF with simple tools like GATK SelectVariants, VCFTools, etc.). I'm not too worried about the redundant data storage for now; our VCF files are really quite small.